pgbouncer/manifest.json

```diff
@@ -10,6 +10,7 @@
     "linux",
     "mac_os"
   ],
+  "package_deps": false,
```

Comment:
In my understanding, even if it's optional, it's a new option that we hadn't considered in previous manifest versions. Although that wasn't really formally documented anywhere (we should start doing that), so we can maybe get away with it this time.